caller/cwe_checker.ml

```diff
@@ -142,22 +140,37 @@ let process_input () : string * string list =
       if check_for_module_versions input then exit 0;
       check_for_no_logging input;
       let binary_path = check_for_binary_path input in
-      let split_flags = List.partition_tf input ~f:(fun x -> (String.is_prefix x ~prefix:"-config") || (String.is_prefix x ~prefix:"-out") || (String.is_prefix x ~prefix:"-partial")) in
+      let split_flags = List.partition_tf input ~f:(fun x -> (String.is_prefix x ~prefix:"-config") || (String.is_prefix x ~prefix:"-out")
+                                                             || (String.is_prefix x ~prefix:"-partial") || (String.is_prefix x ~prefix:"-api")) in
       let flags = remove_element (snd split_flags) binary_path in
-      let params = fst split_flags in
-      check_partial params; check_config params; check_output_path params;
-      (binary_path, params @ process_flags flags)
+      let input_params = fst split_flags in
+      let params = List.map cmdline_params ~f:(fun param -> match param with | (p, _) -> "-" ^ p) in
+      check_params params input_params;
+      (binary_path, input_params @ process_flags flags)
     )
 
 
+let setup_command (bin_path : string) (args : string list) : string =
+  let api = List.exists args ~f:(fun arg -> String.is_prefix arg ~prefix:"-api") in
+  let bare_command = match api with
+  | true -> "bap " ^ bin_path ^ " --pass=cwe-checker,api "
```

Comment:
Is mentioning `api` here actually needed? Since it is now a dependency of the cwe_checker plugin, the api plugin should run anyway.

Reply:
I don't know if I'll be able to call the --api-path param otherwise